infra/modules/providers/azure/provider/README.md

```diff
@@ -0,0 +1,23 @@
+# Azure Provider
+
+## Summary
+
+This module will configure a provider and install required packages mentioned below, needed by terraform to deploy Azure infrastructure resources.
+
+## Packages
+
+### Azure Resource Manager (Version 1.23.0 or higher)
+
+Azure Resource Manager enables you to repeatedly deploy your app and have confidence your resources are deployed in a consistent state. In Cobalt, we use Azure Resource manager extensively for deployment of our resources using Terraform
+
+### Terraform (Version 0.11.13 or higher)
+
+Terraform enables you to safely and predictably create, change, and improve infrastructure. It is an open source tool that codifies APIs into declarative configuration files that can be shared amongst team members, treated as code, edited, reviewed, and versioned. In Cobalt, we use Terraform for all our resource deployments on Azure.
+
+### Azure Active Directory
+
+Azure Active Directory provides reliability and scalability one needs with identity services that work with on-premises, cloud, or hybrid environment. In Cobalt, Azure Active directory is used for the traffic manager role assignment.
+
```

Comment:
@manojvazirani please take a closer look at the example `REAMDE` I mentioned in my previous note. It would add more context for cobalt developers to include things like module usage, a bullet list of module goals, author, license, prerequisites, etc.

Reply:
I agree on adding those for the actual modules which outputs as a resource but do we need to add those for provider module as they wont have any test scenarios. If we go that path, we should define a template for all the Readme's as a requirement for all the modules to be based on this:
https://github.com/Azure/terraform-azurerm-loadbalancer/blob/master/README.md